app/code/Transoft/Blog/view/adminhtml/requirejs-config.js

```diff
@@ -0,0 +1,7 @@
+var config = {
+    map: {
+        '*': {
+            'Magento_Ui/view/base/web/js/form/element/image-uploader': 'Transoft_Blog/js/form/element/image-uploader'
```

Comment:
Why do we need to replace **core** image uploader?

Reply:
removed